app/model/persisted/assessor/AssessorAvailability.scala

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
We did, but I've created the assessor package and moved the case classes to their own files.